### ISSUE TITLE
Browserify bundler fails

### ISSUE THREAD

**Issue by AzuStar** [konvajs/konva#1166]
Hi,
Used latest Konva@7 and decided to update in favour of autodraws introduced in Konva@8.
Since the update, browserify fails with parse error:
`'import' and 'export' may appear only with 'sourceType: module'`

I have obviously tried googling all cases, but I assume from similar cases, it all fails because node-konva natively requires .eslintrc with `sourceType: module` flag under parser options, so that full-ES Konva would still be transpilable to CommonJS.

My set up
```

	"devDependencies": {
		"@types/fs-extra": "^9.0.12",
		"@types/glob": "^7.1.4",
		"@types/mocha": "^7.0.2",
		"@types/node": "^13.13.52",
		"@types/vscode": "^1.58.1",
		"@typescript-eslint/eslint-plugin": "^2.30.0",
		"@typescript-eslint/parser": "^2.30.0",
		"eslint": "^6.8.0",
		"glob": "^7.1.7",
		"konva": "^8.1.3",
		"mocha": "^7.1.2",
		"typescript": "^3.9.10",
		"vscode-test": "^1.6.1",
		"browserify": "^17.0.0",
		"tsd": "^0.14.0"
	}
```

The error is thrown on index.js first line:
Import Konva from '_Internals'.

**Comment by lavrton** [comment]
I just released konva@8.2.0 that has CommonJS modules. Please try to use it https://github.com/konvajs/konva#commonjs-modules

Let me know if it doesn't work.

**Comment by AzuStar** [comment]
Suggestion by @fromaline worked like a charm.
Tested CommonJS modules, same as above - worked like a charm.
